Client/core/CModelCacheManager.cpp

```diff
@@ -229,13 +255,15 @@ void CModelCacheManagerImpl::DoPulse()
     //  128MB streaming = 8+8 MB for peds & vehicles        36 peds + 56 veh
     //  256MB streaming = 16+8 MB for peds & vehicles       72 peds + 56 veh
     //
-    int iStreamingMemoryAvailableKB = *(int*)0x08A5A80;
-
-    SSamplePoint<float> pedPoints[] = {{65536, 9}, {98304, 18}, {131072, 36}, {262144, 72}};
-    SSamplePoint<float> vehPoints[] = {{65536, 7}, {98304, 28}, {131072, 56}, {262144, 56}};
-
-    m_uiMaxCachedPedModels = (int)EvalSamplePosition<float>(pedPoints, NUMELMS(pedPoints), (float)iStreamingMemoryAvailableKB);
-    m_uiMaxCachedVehicleModels = (int)EvalSamplePosition<float>(vehPoints, NUMELMS(vehPoints), (float)iStreamingMemoryAvailableKB);
+    const auto iStreamingMemoryAvailableKB = *(int*)0x08A5A80;
```

Comment:
You shouldn't access game memory directly in the **core** project. Also, why not cast the value to **float** here?

Reply:
I didn't write this code, it was like this.